google/cloud/forseti/notifier/notifiers/inventory_summary.py

```diff
@@ -161,6 +167,26 @@ def transform_to_template(data):
             template_data.append(dict(resource_type=key, count=value))
         return sorted(template_data, key=lambda k: k['resource_type'])
 
+    @staticmethod
+    def _get_gsuite_dwd_status(summary_data):
```

Comment:
If you wan to stick with boolean return, then the name of this method would be better with a boolean naming:
```
def _is_gsuite_dwd_enabled(summary_data):
```

But I think, it's simpler if you just return `enabled` or `disabled` directly from this method.  Which will make the code above simpler: saves a dict and a call to the dict.  In this case, the method name would stay as is.